README.md

```diff
@@ -63,20 +63,46 @@ Add to _course.json_ under _\_globals.\_extensions_.
 
 >> **text** (string): The text of the tooltip
 
-Add to _components.json_, _blocks.json_ or _articles.json_.
+Add to _course.json_.
 
 **\_pageLevelProgress** (object): The Page Level Progress object that contains a value for **\_isEnabled**.
 
 > **\_isEnabled** (boolean): Turns **Page Level Progress** on and off. Acceptable values are `true` and `false`.
 
+> **title** (string): Alternate course title to display when using `_showAtCourseLevel: true`.
```

Comment:
The code does not reflect this restriction.

Reply:
I just meant that the course title won't appear at all if `_showAtCourseLevel` isn't `true`. Does this bit need to be reworded?

Reply:
Hmm, nope, it's fine. I read it out of context. 👍